docs/get-started.md

```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
 
 ### Step 1 Install Nebula Graph
 
-The easiest way to startup `nebula` is using `docker`.
+The easiest way to startup `Nebula` is using `Docker`.
```

Comment:
no backtick around **Docker** and **Nebula**

Reply:
Thanks for your patience review and professional suggestions, I've learned a lot. Have fixed all the backticks and expressions.